src/react-native/xcode.ts

```diff
@@ -142,17 +157,73 @@ export function addSentryWithBundledScriptsToBundleShellScript(
   if (isLikelyExpoScript) {
     const SENTRY_REACT_NATIVE_XCODE_PATH =
       "`\"$NODE_BINARY\" --print \"require('path').dirname(require.resolve('@sentry/react-native/package.json')) + '/scripts/sentry-xcode.sh'\"`";
-    return script.replace(
+    patchedScript = script.replace(
       /^.*?(packager|scripts)\/react-native-xcode\.sh\s*(\\'\\\\")?/m,
       // eslint-disable-next-line no-useless-escape
       (match: string) => `/bin/sh ${SENTRY_REACT_NATIVE_XCODE_PATH} ${match}`,
     );
   }
 
-  return script;
+  if (patchedScript === script) {
+    // No changes were made
+    clack.log.error(
+      `Failed to patch ${chalk.cyan(
+        'Bundle React Native code and images',
+      )} build phase.`,
+    );
+    if (isLikelyExpoScript) {
+      return new ErrorPatchSnippet(
+        makeCodeSnippet(true, (unchanged, plus, _minus) => {
+          return unchanged(
+            `${plus(
+              `/bin/sh \`"$NODE_BINARY" --print "require('path').dirname(require.resolve('@sentry/react-native/package.json')) + '/scripts/sentry-xcode.sh'"\``,
+            )}\`"$NODE_BINARY" --print "require('path').dirname(require.resolve('react-native/package.json')) + '/scripts/react-native-xcode.sh'"\``,
```

Comment:
L: It doesn't break any functionality, but it looks better when printed out with the space in between.

````suggestion
            )} \`"$NODE_BINARY" --print "require('path').dirname(require.resolve('react-native/package.json')) + '/scripts/react-native-xcode.sh'"\``,
````